cms/envs/aws.py

```diff
@@ -434,6 +434,9 @@
 # OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
 OAUTH_OIDC_ISSUER = ENV_TOKENS['OAUTH_OIDC_ISSUER']
 
+#### JWT configuration ####
+JWT_AUTH.update(ENV_TOKENS.get('JWT_AUTH', {}))
```

Comment:
`JWT_AUTH` is part of edxapp's [`generic_env_config`](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L842), so it should be available to Studio at startup.